install_scripts/templates/swarm/init.sh

```diff
@@ -212,7 +212,7 @@ stackDeploy() {
     fi
 
     # Update replicated to the new version.
-    docker stack deploy -c /tmp/replicated-docker-compose.yml "$SWARM_STACK_NAMESPACE"
+    docker stack deploy --with-registry-auth -c /tmp/replicated-docker-compose.yml "$SWARM_STACK_NAMESPACE"
```

Comment:
This works if there is no auth file?

Reply:
i think so. this seems like a flag that should be opt out rather than opt in